src/components/button/themes/button/light/button.bootstrap.scss

```
@@ -16,15 +16,15 @@ $outlined-theme: digest-schema($outlined-schema);
     @include css-vars($flat-theme);
 
     [part='base'] {
-        @include border-radius(var-get($flat-theme, 'border-radius'));
+        @include border-radius(map.get($flat-theme, 'border-radius')...);
```

This way of implementing it completely removes the ability for the user to set a custom border-radius via the CSS variable.

I believe the following should still work.
```suggestion
        @include border-radius(var-get($flat-theme, 'border-radius')...);
```